cmake-format-rapids-cmake.json

```diff
@@ -88,6 +88,24 @@
         }
       },
 
+      "rapids_cpm_cuco": {
```

Comment:
Some of this diff will be reduced by #366. I fixed some places where things weren't being sorted consistently. I include that diff in this PR to avoid conflicts once #366 is merged.